### PR TITLE
Reorder and change all default styles with keys;

### PR DESCRIPTION
**Pull request by ye4241**
## Describe the changes you have made to improve this project

- Sort ResourceDictionary and Styles by name in Generic.xaml and Controls.xaml;
- Add all x:Key for all controls and  add default styles in Generic.xaml and Controls.xaml;
- Change all TargetType to x:Type style;

## Unit test

None.

## Additional context

- ~Remove political content from README.md;~

## Closed Issues

None.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/MahApps/MahApps.Metro?pullRequest=4290) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/MahApps/MahApps.Metro?pullRequest=4290) before we can accept your contribution.<br/><br/>:x: ye4241 [sign now](https://cla.dotnetfoundation.org/MahApps/MahApps.Metro?pullRequest=4290)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/MahApps/MahApps.Metro?pullRequest=4290) it.</sub>

**Comment by Nuklon** [comment]
Remove political content from README.md;
> ye4241, China

What a surprise.

File: readme.md [hunk 1]
```diff
@@ -1,5 +1,3 @@
-[![Stand With Ukraine](https://raw.githubusercontent.com/vshymanskyy/StandWithUkraine/main/banner2-direct.svg)](https://vshymanskyy.github.io/StandWithUkraine)
-
```
Comment (punker76) [inline readme.md:2]: Please revert this change. Thx

**Review by punker76 — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (ye4241) [reply readme.md]: Reverted.

**Pull request closed without merging**